recipe/symfony-flex.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Deployer;
+
+require_once __DIR__ . '/symfony3.php';
+
+/**
+ * Symfony Flex Configuration
+ */
+
+//No need to clear anything
+set('clear_paths', []);
```

Comment:
It's already define in symfony3 recipe :)

Reply:
Nope

Reply:
My bad ! Sorry !